icalevents/icalparser.py

```diff
@@ -313,8 +329,11 @@ def parse_events(content, start=None, end=None, default_span=timedelta(days=7)):
             # use it; otherwise, attempt to load the rules from pytz.
             start_tz = None
             end_tz = None
+            if e.all_day and e.recurring:
```

Comment:
Was the old logic for all day events wrong?
Now we're only applying this on recurring all day events.

If it's not applied to all "all day" events then the comment below in this block (L337-344) isn't true.

Reply:
you are right! it is not needed at all. All day events are in a timezone.